sources/packages/web/src/components/institutions/modals/EditInstitutionUserModal.vue

```diff
@@ -174,10 +196,24 @@ export default {
     };
     const getInstitutionUserDetails = async () => {
       try {
-        const respUser = await InstitutionService.shared.getInstitutionLocationUserDetails(
+        userData.value = await InstitutionService.shared.getInstitutionLocationUserDetails(
           props.institutionUserName,
         );
-        userData.value = respUser;
+
+        const adminAuth = userData.value.authorizations?.find(
```

Comment:
if `authorizations` is not there `undefined.find(` will throw error right?

Reply:
Undefined.find will throw an error. 
When Authorizations are not there, we want adminAuth to be undefined.

Reply:
Resolving as per discussion.